wagtail/contrib/search_promotions/models.py

```diff
@@ -39,6 +55,9 @@ def garbage_collect(cls):
         """
         Deletes all Query records that have no daily hits or editors picks
         """
+        # if the related_name "editors_picks" is in this Query instance:
+        # use a comprehension of a single kwargs element to filter queries
+        # without editors_picks
```

Comment:
```suggestion
```

I understand the desire to add this, it can be a bit tricky at first glance to understand what this code is doing.

I recommend you raise a single PR that adds this content but puts it into the docstring possibly.

For now, it can be removed from this PR.